src/main/java/baseball/controller/GameController.java

```diff
@@ -0,0 +1,72 @@
+package baseball.controller;
+
+import baseball.domain.*;
+import baseball.view.OutputView;
+
+
+public class GameController {
+
+    private final Game game;
+    private final Player player;
+    private final Computer computer;
+
+    public GameController(Game game, Player player, Computer computer) {
+        this.game = game;
+        this.player = player;
+        this.computer = computer;
+    }
+
+    public void run() {
+        initGame();
+        do {
+            playGame();
+            String command = checkRestartWhenMaxStrike();
+            setActionOfRestartAndExit(command);
+        } while (!game.isExitStatus());
+    }
+
+    public void initGame() {
+        OutputView.printInit();
+    }
+
+    public void playGame() {
+        OutputView.printInput();
+        player.setInputBalls();
+
+        playBaseBall();
+        OutputView.printGameResult(game);
+    }
+
+    private void playBaseBall() {
+        int strikeCount = Referee.getStrikeCount(player.getBalls(), computer.getBalls());
+        int ballCount = Referee.getBallCount(player.getBalls(), computer.getBalls());
+        game.setGameResult(strikeCount, ballCount);
+    }
```

Comment:
```suggestion
    public void playGame() {
        OutputView.printInput();
        player.setInputBalls();

        game = new Game(player.getBalls(), computer.getBalls());
        OutputView.printGameResult(game);
    }

```

playGame() -> playBaseBall()에서 세터메소드인  game.setGameResult() 를 호출하여 game을 새로 세팅 해주는것 대신
playGame()에서 game을 새로 만드는건 어떨까요?

getStrikeCount, getBallCount 의 메소드들을 Game 클래스의 private 메소드로 선언한 다음에  
Game 클래스의 생성자에서 getStrikeCount, getBallCount를 이용해 strike와 ball를 구하면 어떨까 싶습니다.

Reply:
오 좋은 방법인것 같네요! 코드도 더 깔끔해지구요.